aten/src/ATen/TensorIterator.cpp

```diff
@@ -1306,6 +1356,11 @@ void TensorIterator::set_output(int64_t output_idx, IntArrayRef sizes, IntArrayR
   }
 }
 
+// Not actually used by anything, here for completeness
```

Comment:
might make the design clearer to just omit it

Reply:
Can't, it's a pure virtual method and I don't implement it then the class is abstract

Reply:
ah ok then I'd elaborate the comment, if possible in like a sentence or two - i.e. why it is that `maybe_get_output()` isn't part of the control flow at all when `TensorIterator` is the leaf class.